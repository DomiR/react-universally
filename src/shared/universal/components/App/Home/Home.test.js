/* eslint-disable no-undef */
/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Home from './Home';

describe('<Home />', () => {
  test('The component renders and does not break.', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper.find('article').length).toBe(1);
  });
  test('renders a snapshot', () => {
    const wrapper = shallow(<Home />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
