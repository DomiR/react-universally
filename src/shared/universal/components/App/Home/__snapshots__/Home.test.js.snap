exports[`<Home /> renders a snapshot 1`] = `
<article>
  <HelmetWrapper
    title="Home" />
  <p>
    This starter kit contains all the build tooling and configuration you need to kick off your next universal react project, whilst containing a minimal project set up allowing you to make your own architecture decisions (redux/mobx etc).
  </p>
</article>
`;
